beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4397,12 +4397,6 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             if cached_head.head_block_root() == parent_block_root {
                 (Cow::Borrowed(head_state), cached_head.head_state_root())
             } else {
-                info!(
-                    self.log,
-                    "Missed snapshot cache during withdrawals calculation";
-                    "slot" => proposal_slot,
-                    "parent_block_root" => ?parent_block_root
-                );
```

Comment:
I deleted this as it wasn't accurate. The state cache kicks in below when we call `get_advanced_hot_state`. If it misses, it will log a warning from the store.

Reply:
Oh yeah, thanks!